### PR TITLE
[travis] Add TrafficLight, I2C, and WebBT samples as standard builds

### PR DESCRIPTION
**Pull request by grgustaf**
Signed-off-by: Geoff Gustafson <geoff@linux.intel.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,10 +31,15 @@ install: >
...
+  source zjs-env.sh $PARTSIZE && make update && source deps/zephyr/zephyr-env.sh
 
 script: >
-  if [ "$TARGET" == "default" ]; then
-    make V=1; else
-    make V=1 "${TARGET}";
+  if [ "$TARGET" == "zephyr" ]; then
+    make V=1
+    make V=1 "JS=samples/TrafficLight.js"
+    make V=1 "JS=samples/I2C.js"
```
Comment (baleboy) [inline .travis.yml]: Testing each JS sample could be done in parallel on a separate VM, but is that overkill?

**Comment by baleboy** [comment]
+1 with some optional comments

**Comment by grgustaf** [comment]
Actually, I started out with them all in parallel (still visible in first commits) but then saw that we are limited to like 4-5 VMs at a time unless we pay for an account. That would have doubled the time to complete the tests. Most of the overhead is in getting the VM ready so I think it makes sense to run multiple tests in one VM.

As for K64F, that will make sense at some point when more tests are compatible, but for right now I have special K64F versions of some samples. So yes at some point.

**Comment by grgustaf** [comment]
@baleboy, oh hmm should I put && in between my make commands so that any of them will result in the test failing? (Bash isn't my native tongue.)

**Comment by baleboy** [comment]
@grgustaf actually now that you ask I'm surprised it even works without && - clearly I'm no bash expert myself. Are you sure it's not running only the first command? I scanned through the log but couldn't figure it out.

**Comment by baleboy** [comment]
To clarify, afaik multi-line if blocks are fine in bash, but I thought Travis' "Yaml-to-bash" conversion was handling everything as a one-line command. But if it works, I think a failure in any of the statements will cause the whole build to fail.

**Comment by baleboy** [comment]
I can see the line "Creating C string from JS application" three times in the build log, so it looks everything went fine.

**Comment by grgustaf** [comment]
OK right, and I remember the build took longer so I know it's running all three. But maybe the best way to know for sure is to purposely break one of the early builds.

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**